src/cascadia/TerminalSettingsEditor/AddProfile.h

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.

```

Comment:
nit: add an abstract and plop your name on it.

I know some of the other pages don't have it. They should. The reason they don't is because we were rushing to get this thing out there haha.